src/App.tsx

```diff
@@ -15,36 +15,81 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  default = '',
+  alphabet = 'Sort alphabetically',
+  length = 'Sort by length',
+}
+
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState<SortType>(SortType.default);
+  const [reversed, setReversed] = useState(false);
+
+  const visibleGoods = [...goodsFromServer];
+
+  if (sortField) {
```

Comment:
Our mentors mentioned that better create separate function for all filters and this function return filtered array with goods